packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -191,7 +191,7 @@ export function TableNG(props: TableNGProps) {
         },
         ...(footerOptions?.show && {
           renderSummaryCell() {
-            return <>{getFooterValue(fieldIndex, footerItems)}</>;
+            return <>{getFooterValue(fieldIndex, footerItems, false, justifyColumnContent)}</>;
```

Comment:
Shouldn't isCountRowsSet? be dependent on calculation option?

For calculation "Count", when `countRows` is enabled.